taipy/gui/builder/_element.py

```diff
@@ -65,14 +107,51 @@ def _parse_property_key(key: str) -> str:
             return f"{match.group(1)}[{match.group(2)}]"
         return key
 
-    @staticmethod
-    def _parse_property(value: t.Any) -> t.Any:
+    def _parse_property(self, key: str, value: t.Any) -> t.Any:
         if isinstance(value, (str, dict, Iterable)):
             return value
+        if isinstance(value, FunctionType):
+            if key.startswith("on_"):
+                return value
+            else:
+                try:
+                    st = ast.parse(inspect.getsource(value).strip())
+                    lambda_fn = next((node for node in ast.walk(st) if isinstance(node, ast.Lambda)), None)
+                    if lambda_fn is not None:
+                        args = [arg.arg for arg in lambda_fn.args.args]
+                        targets = [
+                            compr.target.id  # type: ignore[attr-defined]
+                            for node in ast.walk(lambda_fn.body)
+                            if isinstance(node, ast.ListComp)
+                            for compr in node.generators
+                        ]
+                        tree = _TransformVarToValue(self.__calling_frame, args + targets + python_builtins).visit(
+                            lambda_fn
+                        )
+                        ast.fix_missing_locations(tree)
+                        if sys.version_info < (3, 9): # python 3.8 ast has no unparse
+                            string_fd = io.StringIO()
+                            _Unparser(tree, string_fd)
+                            string_fd.seek(0)
+                            lambda_text = string_fd.read()
+                        else:
+                            lambda_text = ast.unparse(tree)
+                        new_code = compile("new_lambda = " + lambda_text, "<ast>", "exec")
```

Comment:
Could make new_lambda a const

Reply:
How do you mean ?

Reply:
the `new_lambda` is used several times as a namespace variable below. we could make that into a const.